Cargo.dev.toml

```diff
@@ -0,0 +1,96 @@
+cargo-features = ["resolver"]
+
+[workspace]
+members = [
+	"control",
+	"signal",
+	"flow",
+	"sense",
+	"traits",
+]
+
+resolver = "2"
+
+[profile.dev]
+split-debuginfo = "unpacked"
+
+[patch.'https://github.com/paritytech/substrate']
```

Comment:
Do we actually need those `patch` items here? As every module in this project contains specific list of required dependencies with correct versions. Maybe I am not getting it, but why do we need to list all the modules here, which are never used by any of pallets in this project.

Reply:
yes, if you want a cargo file and build which is compatible to zero-network? otherwise hardwired and ten places to edit?